jsapp/js/featureFlags.ts

```diff
@@ -0,0 +1,108 @@
+/**
+ * In URL use value, not key of enum.
+ * For our sanity, use camel case and match key with value.
+ */
+export enum FeatureFlag {
+  activityLogsEnabled = 'activityLogsEnabled',
+}
+
+/**
+ * This function reads query parameters and processes them to find feature flags
+ * - The query parameters are read from the URL
+ * - Recommended naming convention: `ff_FeatureNameEnabled`
+ * - Any parameter starting with "ff_" is considered to be a feature flag
+ * - A feature flag parameter needs to have its value equals "true" to be set to true
+ * - Enabled flags are stored on sessionStorage by its name **without the ff_**
+ * - Disabled (false) flags are removed from the sessionStorage object
+ * - A flag won't disable itself unless it's set to false or the sessionStorage object is deleted (manually or by session ending)
+ *
+ *  Uses:
+ * - Destructuring
+ *  ```js
+ *    # https://kf.kobotoolbox.org/#/projects/home?ff_FeatureNameEnable=true&ff_FeatureNameEnable2=true
+ *    const {FeatureNameEnabled} = getFeatureFlags()`
+ *    console.log(FeatureNameEnabled) # true
```

Comment:
oops, wrong comment char. also more below.
```suggestion
 *    console.log(FeatureNameEnabled) // true
```